### PR TITLE
LPS-161324 Remove SantizerUtil from modules

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from #1528

### CODE REVIEW

**Comment by julschong** [comment]
pushed to fix SF on persistence_impl.ftl

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:stop

**Comment by julschong** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@julschong no regressions found.

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl [hunk 1]
```diff
@@ -2937,6 +2966,10 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 			</#if>
 			protected FinderCache finderCache;
 		</#if>
+		<#if sanitizeTuples?size != 0 && serviceBuilder.isVersionGTE_7_4_0() && dependencyInjectorDS>
+			@Reference
+			protected volatile List<Sanitizer> sanitizers;
```
Comment (tinatian) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:2971]: @julschong It does not have to be **protected**, please change it to private

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl [hunk 2]
```diff
@@ -2937,6 +2966,10 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 			</#if>
 			protected FinderCache finderCache;
 		</#if>
+		<#if sanitizeTuples?size != 0 && serviceBuilder.isVersionGTE_7_4_0() && dependencyInjectorDS>
+			@Reference
+			private volatile List<Sanitizer> sanitizers;
```
Comment (julschong) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:2971]: pushed and modified protected to private

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine, I'm trigger ci again to make sure this change is ok

**Comment by julschong** [comment]
@vicnate5
Hi Victor, I just noticed Catarine is off today.
could you please help me with some of my pulls?
Thanks!

**Comment by vicnate5** [comment]
@Kvale1733 can you check this one?

**Comment by Kvale1733** [comment]
@julschong 
Regression:

com.liferay.headless.delivery.resource.v1_0.test.BlogPostingResourceTest
`ava.lang.AssertionError: expected:<2> but was:<1>  at org.junit.Assert.fail(Assert.java:89)`[Testray](https://testray.liferay.com/home/-/testray/case_results/1969714467)

**Comment by julschong** [comment]
@Kvale1733 Hi Kevin, I tested locally and passed
lemme rebase and run ci again

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:stop

**Comment by julschong** [comment]
pushed to include newly added SanitizerUtil usage

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@julschong No regressions found.

**Comment by tinatian** [comment]
@julschong As we talked, please resend.

Thanks.
Tina

**Pull request closed without merging**